ubuntudesign/www.ubuntu.com
# Iot

## Done

On the IoT section:
 - Included Graham's initial full-width work
 - fixed the obvious styling bugs
 - removed some styling that will be full corrected with later designs
 - removed "features" section

## QA

Click around the entire /internet-of-things section and ensure that it displays correctly.
